src/components/application_manager/src/request_controller.cc

```diff
@@ -190,8 +190,9 @@ RequestController::TResult RequestController::addHMIRequest(
 
   const uint64_t timeout_in_mseconds =
       static_cast<uint64_t>(request->default_timeout());
-  RequestInfoPtr request_info_ptr(
-      new HMIRequestInfo(request, timeout_in_mseconds));
+  RequestInfoPtr request_info_ptr =
+      utils::MakeShared<HMIRequestInfo>(request, timeout_in_mseconds);
+  request_info_ptr->set_hmi_level(mobile_apis::HMILevel::INVALID_ENUM);
```

Comment:
@vlantonov it's not clear for me how these changes relate to core crash on USB unplug. Maybe add some explanation to PR description?